test/models/offerCancel.ts

```diff
@@ -0,0 +1,64 @@
+import { ValidationError } from 'ripple-api/common/errors'
+import { verifyOfferCancel } from './../../src/models/transactions/offerCancel'
+import { assert } from 'chai'
+
+/**
+ * OfferCancel Transaction Verification Testing
+ *
+ * Providing runtime verification testing for each specific transaction type
+ */
+describe('OfferCancel Transaction Verification', function () {
+
+    it (`verifies valid OfferCancel`, () => {
+        const offer = {
+            Account: "rnKiczmiQkZFiDES8THYyLA2pQohC5C6EF",
+            Fee: "10",
+            LastLedgerSequence: 65477334,
+            OfferSequence: 60797528,
+            Sequence: 60797535,
+            SigningPubKey: "0361BFD43D1EEA54B77CC152887312949EBF052997FBFFCDAF6F2653164B55B21...",
+            TransactionType: "OfferCancel",
+            TxnSignature: "30450221008C43BDCFC68B4793857CA47DF454C07E5B45D3F80E8E6785CAB9292...",
+            date: "2021-08-06T21:04:11Z"
+        } as any
```

Comment:
Why do we cast the offer as any? (Curious)

Reply:
Otherwise, TypeScript would complain that it's not the right format. But we want it to check at runtime instead of at compile time, so we cast it as `any`, which basically tells TS to ignore this.